src/App.tsx

```diff
@@ -1,15 +1,33 @@
 import React from 'react';
 
+import { PostList } from './components/PostList/PostList';
+
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { Post } from './types/Post';
+
+const preparedPosts: Post[] = posts.map((post) => ({
+  ...post,
+
+  comments: comments.filter((comment) => (
+    comment.postId === post.userId
+  )) || [],
```

Comment:
Unnecessary ||, filter will return an empty array (which, when converted to boolean is true) if no elements would pass the check
```suggestion
  comments: comments.filter((comment) => (
    comment.postId === post.userId
  )),
```